app/controllers/sessions_controller.rb

```diff
@@ -34,6 +34,11 @@ def create
     redirect_to session[:previous_url] || root_url
   end
 
+  def update
```

Comment:
I'm not sure I like the name of this route – updating a session constitutes accepting the ToS? It might be more descriptive if we forced the route to have a different name instead (like `accept_tos`).